progress/README.md

```diff
@@ -2,11 +2,12 @@
 
 ## Requests
 
-|                     Route                     |                                          Description                                          |
-| :-------------------------------------------: | :-------------------------------------------------------------------------------------------: |
-|     [POST /progresses](#post-progresses)      |                                 Creates a new progress entry.                                 |
-|      [GET /progresses](#get-progresses)       |                 Retrieves progress entries based on the provided parameters.                  |
-| [GET /progresses/range](#get-progressesrange) | Retrieves the progress records for a particular user or task within the specified date range. |
+|                     Route                     |                                           Description                                           |
+| :-------------------------------------------: | :---------------------------------------------------------------------------------------------: |
+|     [POST /progresses](#post-progresses)      |                                  Creates a new progress entry.                                  |
+|      [GET /progresses](#get-progresses)       |                  Retrieves progress entries based on the provided parameters.                   |
+| [GET /progresses/range](#get-progressesrange) |  Retrieves the progress records for a particular user or task within the specified date range.  |
+| [GET /progresses/:type/:typeId/date/:date](#get-progressestypetypeiddatedate)  | Retrieves the progress documents based on a specific user ID or task ID for the specified date. |
 
 ## POST /progresses
```

Comment:
what if we only provide type and typeId and don't provide date, then what will be response

Reply:
No you will not get a response as this path would not match and you will get an error not found